src/modules/pair/pair.resolver.ts

```diff
@@ -209,6 +209,26 @@ export class PairResolver {
             : undefined;
     }
 
+    @ResolveField()
+    async hasFarms(@Parent() parent: PairModel): Promise<boolean> {
```

Comment:
the other functions have different formatting (`@Parent ...` is on a new line). maybe keep consistency

Reply:
This is due to prettier rules (print width is currently set to 80 chars). Formatting happens automatically on each "save"